cgi-bin/DW/Controller/Support/Faq.pm

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Support::Faq
+#
+# This controller is for the Support FAQ page.
+#
+# Authors:
+#      hotlevel4 <hotlevel4@hotmail.com>
+#
+# Copyright (c) 2015 by Dreamwidth Studios, LLC.
+#
+# This is based on code originally implemented on LiveJournal.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself.  For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+
+package DW::Controller::Support::Faq;
+
+use strict;
+use warnings;
+use vars qw(%GET);
```

Comment:
On second thought, I think that this doesn't actually do anything. It initializes the %GET variable but doesn't actually map it to the get arguments (the way that BML does).

You can do that by following the example code I listed above.

*However* looking at the code, this is pretty much a duplicate of the faqbrowse functionality, so I'd rather just remove it entirely.